src/sync/index.js

```diff
@@ -555,7 +556,8 @@ async function updateOracleBalance(oracleAddress, topicSet, db) {
     }
   }
 
-  const balances = _.map(value[0].slice(0, oracle.numOfResults), balanceBN => balanceBN.toJSON());
+  const balances = _.map(value[0].slice(0, oracle.numOfResults),
+    balanceBN => Web3Utils.hexToNumberString(balanceBN.toJSON()));
```

Comment:
should we use `toString` instead of `toJSON` for all

Reply:
Why not? toJSON is an alias for toString(16)

Reply:
why not directly use toString(10)?

Reply:
oh right lol good point. will change.